src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -55,36 +52,24 @@ export default function applyAuthInterface(httpClient, authConfig) {
    * @param route: used to return user after login when not authenticated.
    * @returns Promise that resolves to { authenticatedUser: {...}, decodedAccessToken: {...}}
    */
-  httpClient.ensureAuthenticatedUser = route => httpClient
-    .refreshAccessTokenOnce()
-    .then((response) => {
-      const refreshedAccessToken = httpClient.getDecodedAccessToken();
-
-      if (refreshedAccessToken === null) {
-        // This should never happen, but it does. See ARCH-948 for past research into why.
```

Comment:
This case is now handled inside AccessToken's `handleUnexpectedRefreshFailure` callback. It will be interesting to see if it's a case that continues to happen after this refactor.